MyBus/MyBus/Services/GisService.swift

```diff
@@ -34,19 +34,20 @@ public class GisService: NSObject, GisServiceDelegate {
         print("You tapped at: \(latitude), \(longitude)")
         CLGeocoder().reverseGeocodeLocation(CLLocation(latitude: latitude, longitude: longitude)) {
             placemarks, error in
-            if let placemark = placemarks?.first {
-                let point = RoutePoint()
-                point.latitude = latitude
-                point.longitude = longitude
-                if let street = placemark.thoroughfare, let houseNumber = placemark.subThoroughfare {
-                    let address = "\(street as String) \(houseNumber as String)"
-                    point.address = address
-                }
-                completionHandler(point, nil)
-            }else{
-                completionHandler(nil, error)
+            guard let placemark = placemarks?.first where (placemark.locality == "General Pueyrredón" || placemark.locality == "Mar del Plata" || placemark.locality == "Sierra de los Padres" || placemark.locality == "Batán") else {
```

Comment:
Just as an enhancement, it would be great to lowercase the value inside **placemark.locality** and to compare it with lowercase values such as "general pueyrredón", "mar del plata", etc in order to avoid possible changes in results from geocoding.